sync-core/src/main/java/com/cloudant/mazha/CouchURIHelper.java

```diff
@@ -61,6 +63,17 @@ public URI changesUri(Map<String, Object> query) {
                 this.rootUriString,
                 "_changes"
         );
+
+        //lets find the since parameter
+        if(query.containsKey("since")){
+            Object since = query.get("since");
+            if(!(since instanceof String)){
```

Comment:
Did you confirm that the wrong thing happens if we remove this check and pass a string? That is, we end up with `since="123-blah"` rather than `since=123-blah`.